src/pages/_index/_components/what-is-wasmcloud/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { Section } from '@site/src/pages/_components/section';
+import { SectionTag } from '@site/src/pages/_components/section-tag';
+import { SectionHeading } from '@site/src/pages/_components/section-heading';
+import {
+  Switcher,
+  SwitcherButton,
+  SwitcherContent,
+  SwitcherList,
+} from '../what-is-wasmcloud/switcher';
+import { SectionSubheading } from '@site/src/pages/_components/section-subheading';
+import styles from './what-is-wasmcloud.module.css';
+
+type Props = {};
+
+const INTRO_CONTENT: {
+  tag: React.ReactNode;
+  heading: React.ReactNode;
+  description: React.ReactNode;
+} = {
+  tag: 'What is wasmCloud?',
+  heading: 'Fast-track the development, deployment, and management of distributed applications',
+  description: (
+    <p>
+      wasmCloud is an open source project from the Cloud Native Computing Foundation (CNCF) that
+      enables teams to build polyglot applications composed of reusable Wasm components and run
+      them—resiliently and efficiently—across any cloud, Kubernetes, datacenter, or edge.
+    </p>
+  ),
+};
+
+const SWITCHER_CONTENT: Array<{
+  id: string;
+  title: React.ReactNode;
+  image: string;
+  features: Array<{
+    title: React.ReactNode;
+    description: React.ReactNode;
+    link?: string;
+    linkText?: React.ReactNode;
+  }>;
```

Comment:
These, for some reason, show all highlighted instead of only the active one. Is that right? 
<img width="592" alt="image" src="https://github.com/user-attachments/assets/1df92f8d-893f-40c9-87e6-0895d7ba3e79">